python/mol.cpp

```diff
@@ -176,9 +176,22 @@ void add_mol(py::module& m) {
                      (self.given ? " (" : " (not "), "given)>");
     });
 
+  py::class_<Assembly::Oper>(m, "AssemblyOper")
```

Comment:
Would it make sense to declare those bindings as nested classes of `Assembly` to avoid global name pollution? 
If you doing so you would need to [reorder bindings](https://pybind11.readthedocs.io/en/stable/advanced/misc.html#avoiding-c-types-in-docstrings) to avoid C++ signatures in docstrings.